build/parser.mjs

```diff
@@ -1070,12 +1095,16 @@ class Parser {
    * @param {Item} item the item to add the attribute to
    */
   applyMasterable(item) {
+    if (item.parent) return;
     item.masterable = masterableCategories.categories.includes(item.category);
 
     if (item.type?.includes('Component') || item.category === 'Pets') {
       const regex = new RegExp(masterableCategories.regex);
       item.masterable = regex.test(item.uniqueName);
     }
+
+    // Assume the item is outside the masterable category and remove it completely
+    if (!item.masterable) delete item.masterable;
```

Comment:
i thought we wanted to keep these booleans, not just have them not exist

Reply:
Removing it when it's false would reduce data bloat as outside of things that won't give XP it seems a bit excessive.

For example if I were to pull a resource item I wouldn't need to check if it's masterable because resources don't give mastery points

having said that I can see why it would be something to have for maybe Zaws/K-Guns/K-drive and nodes?

Reply:
yeah, that's why i'd previously stopped deleting fields, at the request of others to make sure if it's possible on the type it stays on the type, since some languages don't consume fields missing nicely

Reply:
fair enough